templates/slurm.txt

```diff
@@ -688,3 +689,13 @@ Order = 20
         Config.Plugin = pico.form.Dropdown
         Config.Entries := {[Value=1], [Value=2], [Value=3], [Value=undefined; Label="Any"]}
         Conditions.Hidden := configuration_slurm_ha_enabled isnt true
+
+        [[parameters Node Healthchecks]]
+        Description = "Section for configuring Node Healthchecks"
```

Comment:
We recently changed the name to Node Health Checks, so we might want to have this reflected in all the customer-facing labels.